.claude/CLAUDE.md

```
@@ -35,38 +55,241 @@
 
 ---
 
-## プロジェクト固有の設定
+## 開発フロー
+
+### michiワークフロー（推奨）
+
+michiがインストールされている場合は、cc-sdd（仕様駆動開発）ワークフローを使用します：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)